src/connectors/range/connectRange.js

```diff
@@ -292,6 +292,59 @@ export default function connectRange(renderFn, unmountFn) {
 
         return nextState;
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`